src/main/java/org/truffleruby/core/module/ModuleNodes.java

```diff
@@ -1840,10 +1841,10 @@ public SetMethodVisibilityNode(Visibility visibility) {
             this.visibility = visibility;
         }
 
-        public abstract DynamicObject executeSetMethodVisibility(VirtualFrame frame, DynamicObject module, Object name);
+        public abstract Object executeSetMethodVisibility(VirtualFrame frame, DynamicObject module, Object name);
```

Comment:
Why change the return type here?

Reply:
Not needed, indeed. I wanted to make it `void` but I think that doesn't work for some reason.

Reply:
The idea is it better expresses that the return value shouldn't be used compared to `DynamicObject`.

Reply:
Inheriting RubyBaseNode allows to have `void` execute* methods, so this is fixed now.